earthfile2llb/interpreter.go

```diff
@@ -1587,6 +1589,26 @@ func (i *Interpreter) handleImport(ctx context.Context, cmd spec.Command) error
 	return nil
 }
 
+func (i *Interpreter) handleProject(ctx context.Context, cmd spec.Command) error {
+	projectVal, err := i.expandArgs(ctx, cmd.Args[0], false, false)
+	if err != nil {
+		return i.wrapError(err, cmd.SourceLocation, "failed to expand PROJECT %s", cmd.Args[0])
+	}
+
+	if !strings.Contains(projectVal, "/") {
```

Comment:
nit: this check is unnessisary as it's covered below in the `if len(parts) != 2` check.

I prefer the error message used here though, maybe just simply do:

```
	parts := strings.Split(projectVal, "/")
	if len(parts) != 2 {
		return i.errorf(cmd.SourceLocation, "unexpected format for PROJECT statement, should be: <organization>/<project>")
	}
```

Reply:
SGTM